### PR TITLE
data/amithimani

### PR DESCRIPTION
**Pull request by amithimani**
## Note to reviewers

First pull request for creating page

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5564"></a>

### CODE REVIEW

**Review by schmelto — DISMISSED** [review]

**Review by khushalarora — APPROVED** [review]
Thank you for contributing  :tada:

File: data/amithimani.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+		"Google Cloud Platform",
+		"Microsoft Azure"
+	],
+
+	"socials": [{
+			"icon": "FaTwitter",
+			"url": "https://twitter.com/amithimani1"
+		},
+		{
+			"icon": "FaLink",
```
Comment (imbeshat) [inline data/amithimani.json:20]: You can use the Linkedin icon here.

Comment (amithimani) [reply data/amithimani.json]: Resolved

File: data/amithimani.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+	"tags": [
+		"Open Source",
+		"DevOps",
+		"Chaos Engineering",
+		"Performance Engineering",
+		"Java",
+		"Google Cloud Platform",
+		"Microsoft Azure"
+	],
+
```
Comment (schmelto) [inline data/amithimani.json:14]:
```suggestion
```

**Pull request merged**